content/security/docs/multiaccount.md

```diff
@@ -94,9 +94,55 @@ The source of the script called by credential_process:
+
+Please note that some of these attributes may not be universally unique, for example two EKS clusters may have identical namespaces, and one cluster may have identically named service accounts across namespaces. So when granting access via EKS Pod Identities and ABAC, it is a best practice to always consider the cluster arn and namespace when granting access to a service account.
+
+###### ABAC and EKS Pod Identities for cross account access
+
+When using EKS Pod Identities to assume roles (role chaining) in other accounts as part of a multi account strategy, you have the option to assign a unique IAM role for each service account that needs to access another account, or use a common IAM role across multiple service accounts and use ABAC to control what accounts it can access.
+
+To use ABAC to control what service accounts can assume a role into another account with role chaining, you create a role trust policy statement that only allows a role to be assumed by a role session when the expected values are present. The following role trust policy will only let a role from the EKS cluster account (account ID 111122223333) assume a role if the `kubernetes-service-account`, `eks-cluster-arn` and `kubernetes-namespace` tags all have the expected value.
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::111122223333:root"
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {
+                "StringEquals": {
+                    "aws:PrincipalTag/kubernetes-service-account": "PayrollApplication",
+                    "aws:PrincipalTag/eks-cluster-arn": "arn:aws:eks:us-east-1:111122223333:cluster/ProductionCluster",
+                    "aws:PrincipalTag/kubernetes-namespace": "PayrollNamespace"
+                }
+            }
+        }
+    ]
+}
+```
+
+When using this strategy it is a best practice to ensure that the common IAM role only has `sts:AssumeRole` permissions and no other AWS access.
 
-Please note that these attributes may not be universally unique, for example two EKS clusters may have identical namespaces, and one cluster may have identically named service accounts across namespaces. So when granting access via EKS Pod Identities and ABAC, it is a best practice to always consider the cluster arn and namespace when granting access to a service account with pod identities.
+It is important when using ABAC that you control who has the ability to tag IAM roles and users to only those who have a strict need to do so. Someone with the ability to tag an IAM role or user would be able to set tags on roles/users identical to what would be set by EKS Pod Identities and may be able to escalate their privileges. You can restrict who has the access to set tags the `kubernetes-` and `eks-` tags on IAM role and users using IAM policy, or Service Control Policy (SCP). 
```

Comment:
This contradicts the note from https://docs.aws.amazon.com/eks/latest/userguide/pod-id-abac.html#pod-id-abac-chaining where it says:

> Tags added to the session through the sts:AssumeRole request take precedence in the case of conflict. For example, assume that Amazon EKS adds a key eks-cluster-name and value my-cluster to the session when EKS assume the customer role. You has also added an eks-cluster-name tag to the IAM role with value my-own-cluster. In this case, the former takes precedence and value for the eks-cluster-name tag will be my-cluster.

Can we reconcile these to avoid confusion.

cc @micahhausler

Reply:
That's not a contradiction. If you're using ABAC with in any way, not just with pod identities you MUST control tagging operations on roles/users such that someone couldn't elevate their own privilege by tagging.

Reply:
but not to say we couldn't improve the docs to help clarify that. What I was trying to communicate is that tag governance is important if you want to use ABAC. The tags set by pod identities can't be overwritten by principal tag values, but someone could tag an arbitrary principal to have equivalent attributes to those set by EKS pod identities.